src/presenter/events-list-presenter.js

```diff
@@ -1,22 +1,24 @@
-import {render, replace} from '../framework/render.js';
+import {render, RenderPosition} from '../framework/render.js';
 import NewPointView from '../view/add-new-point-view.js';
-import PointView from '../view/point-view.js';
 import ListPointView from '../view/list-point-view.js';
-import EditPointView from '../view/edit-point-view.js';
 import NoPointView from '../view/no-point-view.js';
 import InfoView from '../view/trip-info-view.js';
 import SortView from '../view/sort-view.js';
-
+import PointPresenter from './point-presenter.js';
 export default class ListPointPresenter {
   #listContainer = null;
   #pointsModel = null;
 
   #listComponent = new ListPointView();
+  #sortComponent = new SortView();
+  #noPointComponent = new NoPointView();
 
   #listPoints = [];
   #listOffers = [];
   #listDestinations = [];
 
+  #pointPresenter = new Map();
```

Comment:
```suggestion
  #pointPresenters = new Map();
```
Я бы назвал во множественном числе, их же много.